python/source/org/libreoffice/ide/eclipse/python/PythonBuilder.java

```diff
@@ -109,22 +109,23 @@ public void fillUnoPackage(UnoPackage pUnoPackage, IUnoidlProject pUnoPrj) {
     }
 
     /**
-     * Get the Python files that are located in the project
+     * Set the Python files (ie: pPythonFiles) that are located in the project
      * directory or one of its sub-folder.
      *
-     * @param pPythonPrj the project from which to get the Python files
-     * @return a list of all the those Python Files
+     * @param pSourceFolder the source folder
+     * @param pPythonFiles the Python files
+     * @param pUnoPrj the project from which to get the Python files
      */
-    private void getPythonFiles(IFolder sourceFolder, ArrayList<IFile> pythonFiles, IUnoidlProject pUnoPrj) {
+    private void getPythonFiles(IFolder pSourceFolder, ArrayList<IFile> pPythonFiles, IUnoidlProject pUnoPrj) {
```

Comment:
Why the `p` prefix here?

Reply:
If we keep this rule, then the parameters of the methods start with `p`.